tests/authentication.test.js

```diff
@@ -321,7 +335,7 @@ describe('Reset Email', () => {
   // occured an error while updating
   it('errored update of the password ', (done) => {
     const pass = {
-      password: 'sesese'
+      password: 'segsese145'
```

Comment:
This is very "plain text", can you create a `mockData` for password in `tests/data/mockData.js`